test/alternator/conftest.py

```diff
@@ -45,7 +49,14 @@ def pytest_addoption(parser):
     # presence.
     parser.addoption('--omit-scylla-output', action='store_true',
         help='Omit scylla\'s output from the test output')
-
+    parser.addoption('--host', action='store', default='localhost',
```

Comment:
looks like no one of this options are needed 
port, api-port - is not used,

the host looks like don't need (see my moments below)

Reply:
Unneeded options are deleted, however, there is no way to remove the host option.

Reply:
I don't understand why. conftest.py already has the "url" option, which includes the host.

Reply:
Ok, I give up, you can add "host" but please add a better help string, something like:
'communicate with the given host on port 8000 via HTTP. use "--url" instead to specify the port and scheme as well.'